scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -49,52 +67,135 @@ namespace <%= namespace %> {
             _LineItems = new CartLineItems();
         }
 
-        /// <summary>
-        /// Returns a link which can be used to check out this cart's line items via a web browser.
-        /// </summary>
-        /// <param name="note">can be used to create a special note for the merchant for this order</param>
-        /// \code
-        /// // Example that creates a checkout link and opens the url in the Browser
-        /// string linkToCheckout = ShopifyBuy.Client().Cart().GetWebCheckoutLink();
-        ///
-        /// Application.OpenURL(linkToCheckout);
-        /// \endcode
-        public string GetWebCheckoutLink(string note = null) {
-            StringBuilder url = new StringBuilder();
-            bool hasLineItem = false;
+        public void GetWebCheckoutLink(Action<string> success, Action<List<string>, string> failure) {
```

Comment:
it these were separate concepts it could be something like: `var checkout = Checkout.Create(cart); checkout.webUrl();` which i like because it's less implicit